backend/grant/user/models.py

```diff
@@ -245,9 +245,32 @@ def get_userid(self, obj):
         return obj.id
 
 
-user_schema = UserSchema()
-users_schema = UserSchema(many=True)
+self_user_schema = SelfUserSchema()
+self_users_schema = SelfUserSchema(many=True)
+
+
+class UserSchema(ma.Schema):
+    class Meta:
+        model = User
+        # Fields to expose
+        fields = (
+            "title",
+            "social_medias",
+            "avatar",
+            "display_name",
+            "userid"
+        )
+
+    social_medias = ma.Nested("SocialMediaSchema", many=True)
+    avatar = ma.Nested("AvatarSchema")
+    userid = ma.Method("get_userid")
+
+    def get_userid(self, obj):
+        return obj.id
+
 
+user_schema = SelfUserSchema()
+users_schema = SelfUserSchema(many=True)
```

Comment:
user_schema is a SelfUserSchema, not a UserSchema.

As an aside, you could reduce this copy paste by using one schema class, and having them exclude the undesired fields like so:
```
self_user_schema = UserSchema()
self_users_schema = UserSchema(many=True)
user_schema = UserSchema(exclude=['email_address'])
users_schema = UserSchema(exclude=['email_address'], many=True)
```

Reply:
I did try this approach a few commits back, but I realized that you could easily forget to add the exclude when using `UserSchema` as a nested serializer in another schema.

The code duplication seems worth it to me personally.

Reply:
> user_schema is a SelfUserSchema, not a UserSchema.

Yup, good catch. I'll add a test as well